staging/src/k8s.io/apiserver/pkg/server/options/serving_test.go

```diff
@@ -42,7 +42,6 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/apimachinery/pkg/version"
-	"k8s.io/apiserver/pkg/server"
 	. "k8s.io/apiserver/pkg/server"
```

Comment:
drop the dot import in favor of the explicit import

Reply:
Addressed.